src/main/java/org/jabref/gui/util/Theme.java

```diff
@@ -33,46 +43,137 @@
 
     public static final String BASE_CSS = "Base.css";
 
+    private static final int MAX_IN_MEMORY_CSS_LENGTH = 48000; // 48 kilobytes. Base theme is 33k, Dark adds < 4k.
```

Comment:
Can you provide a reference to the docs for that facts?

Reply:
Do you mean, adding to the comment in Theme.java, a reference to how I determined Base.css and Dark.css are 33k and 4k respectively?

Or do you mean, the user guide should document how the Preview handles errors when the CSS is large?

Reply:
No, sorry for beein unclear, just a reference to the java spec/javadoc where the limit is defined, in case that changes in the future

Reply:
No worries! I've enhanced the Javadoc on this field and on the Theme class, and linked it up with references to the files and the jabref documentation for custom themes. Hopefully this is enough to maintain the value? Only other thing I can think to do, is reference it in the Base and Dark CSS files, but I feel that CSS authors shouldn't have to worry about it, as the maximum isn't a hard limit, just a best effort to catch some edge cases in which uncaught exceptions may occur even with file existence checks.